### PR TITLE
[Android] Add missing icons in editor, refactor 'operator'

### LINKED ISSUE #3388: Add suitable icons in the editor
omm

### PR DESCRIPTION
**Pull request by endim8**
Adds missing icons for `Email` & `Website` (not for address fields as it becomes messy), cleans up implementation of `Operator` (not a complex field so can just use InitBlock) ~~& fixes incorrect padding in the editor menu~~
fixes #3388 ~~and #3393~~

dfsfsfs

Signed-off-by: Harry Bond <endim8@pm.me>

### CODE REVIEW

**Review by vng — APPROVED** [review]

File: android/res/layout/fragment_editor.xml [hunk 1]
```diff
@@ -5,7 +5,6 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  android:fitsSystemWindows="true"
```
Comment (biodranik) [inline android/res/layout/fragment_editor.xml:8]: It was a part of the bugfix, why did you remove it? https://github.com/organicmaps/organicmaps/pull/3325

**Review by biodranik — CHANGES_REQUESTED** [review]

File: android/res/layout/fragment_editor.xml (hunk 1 above)
Comment (endim8) [reply android/res/layout/fragment_editor.xml:8]: Ah interesting. For me, it introduced some padding at the top of the editor. (see pr screenshots)

i'll do more testing and find a fix for both cases 👍

Comment (vng) [reply android/res/layout/fragment_editor.xml:8]: So, does editor scrollable with open keyboard with your fix?

Comment (endim8) [reply android/res/layout/fragment_editor.xml:8]: Unfortunately not yet, I'll mark this PR as a draft

**Comment by biodranik** [comment]
Let's merge everything but leave the padding fix for another PR. Maybe you can try to revert #3325 and find a different fix for it, or @arnaudvergnet can suggest something.

**Comment by arnaudvergnet** [comment]
From a quick glance, seems to me `android:fitsSystemWindows="true"` introduces a padding at the top because the statusbar is transparent.

OM makes the statusbar transparent in nearly the whole app (not only the map activity) and manually adds padding and color to the green header to recreate an opaque statusbar look and feel. IMO this is overly complicated and should be changed. The issue is that the whole app is designed around it so changing it might introduce new bugs if not thoroughly tested.

So because the statusbar is considered transparent and thus displayed over the statusbar, the `fitsSystemWindows` adds a padding to the top the size of the statusbar. You can try launching the app on different devices with/without camera notch to test and see how the padding changes with the statusbar size.

Ideally a proper fix would be to make the statusbar opaque in every screen except the map activity (just like I did for the navigation bar).

**Comment by biodranik** [comment]
Thanks for the details, @arnaudvergnet ! I agree with your proposal to make the status bar opaque everywhere except the map. What would be the easiest/cleaner way to do so?

**Comment by arnaudvergnet** [comment]
I'd say we keep the top padding as it is right now and only focus on icons in this PR. I am trying to clean up the statusbar handling in the hopes to fix https://github.com/organicmaps/organicmaps/issues/3444 and https://github.com/organicmaps/organicmaps/issues/3411. I'll try to make the changes gradually to detect regressions faster.

**Comment by endim8** [comment]
Removed the padding changes, so this PR only updates the icons 👍

**Review by biodranik — APPROVED** [review]
